### PR TITLE
Add membership request GitHub template

### LINKED ISSUE #2: Create issue template for GitHub Org membership application

### PR DESCRIPTION
**Pull request by inteon**
closes #2

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/membership-request.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- id: contributions
+  type: textarea
+  attributes:
+    label: List of contributions to the cert-manager project
+    placeholder: |
+      - PRs reviewed / authored
+      - Issues responded to
+      - Slack conversations you helped in
+  validations:
+    required: true
```
Comment (erikgb) [inline .github/ISSUE_TEMPLATE/membership-request.yml:61]:
```suggestion
    required: true

```

Nit

**Review by erikgb — APPROVED** [review]
/lgtm

Good job, @inteon

File: .github/ISSUE_TEMPLATE/membership-request.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    description: GitHub handle of your sponsor
+    placeholder: e.g. @sponsor-2
+  validations:
+    required: true
+- id: contributions
+  type: textarea
+  attributes:
+    label: List of contributions to the cert-manager project
+    placeholder: |
+      - PRs reviewed / authored
```
Comment (maelvls) [inline .github/ISSUE_TEMPLATE/membership-request.yml:57]: Let's avoid `/` whenever we can (and no space after `/`!). What's confusing to me is that "/" is generally used as an [exclusive disjonction](https://en.wikipedia.org/wiki/Slash_(punctuation)) where it doesn't make sense of having two items selected simultanously. Here, `/` indicates that it is PRs that the person has either reviewed or authored, but not both.

```suggestion
      - PRs reviewed or authored
```

**Pull request merged**